src/docs/about.md

```diff
@@ -0,0 +1,186 @@
+---
+title: What is Broccoli?
+---
+
+Broccoli is a simple Javascript build tool, that allows you to configure your build pipeline in Javascript (like you're
+already used to writing), and handles setting up the filesystem state for each transformation that's going to happen.
+
+${toc}
+
+## What is a build tool?
+
+A build tool's job is to take input files (your javascript, css, html, etc), to process them and output the result of
+those transformations, usually into some distributable form. Typically this will involve things like javascript 
+transformations to allow you to write newer syntax that will work in a browser, to use things like Sass for your CSS,
+etc.
+
+Broccoli.js is different to other build tools. You may be used to tools like [Grunt](https://gruntjs.com/) (a task
+runner), [Gulp](https://gulpjs.com/) (streams and pipes) or [Webpack](https://webpack.js.org/) (a module bundler),
+these all operate on different levels to Broccoli.
+
+Broccoli works at the file-system level, it provides a Javascript API to wrap other node tools like
+[Babel](https://babeljs.io/), [Rollup](https://rollupjs.org/) or [Node-Sass](https://github.com/sass/node-sass), that
+operate on a set of input files, and emit output files, that is all. Broccoli has no knowledge of the contents of 
+your files, only input and output directories.
+
+## Thinking in Broccoli
+
+Broccoli at its heart is very simple. It's really just a set of connected file transformations, performed by plugins,
```

Comment:
Maybe something like:

It is really just a set of directories, connected together by a plugin system. This plugin systems allow behavior between two directories to be described. Broccoli then ensures plugins are called at the appropriate time, to ensure the build is succesfull.